src/lib/operators/sort.cpp

```diff
@@ -177,10 +221,9 @@ class Sort::SortImpl : public AbstractReadOnlyOperatorImpl {
     _null_value_rows = std::make_shared<std::vector<RowIDValuePair>>();
   }
 
- protected:
-  std::shared_ptr<const Table> _on_execute() override {
+  std::shared_ptr<std::vector<RowIDValuePair>> sort_one_column(const std::shared_ptr<PosList>& pos_list = nullptr) {
```

Comment:
Please describe what the parameter does, not what type it is

Reply:
Done in 2e4a1e739bcc9286a783cba00336ca8e61325125